src/pages/Admin/Charity/Withdraws/Withdrawer/WithdrawForm/Network/Network.tsx

```diff
@@ -0,0 +1,88 @@
+import { Listbox } from "@headlessui/react";
+import { useEffect } from "react";
+import { useFormContext } from "react-hook-form";
+import { WithdrawValues } from "../types";
+import { useChainsQuery } from "services/apes";
+import Icon, { DrawerIcon } from "components/Icon";
+import QueryLoader from "components/QueryLoader";
+
+export default function Network() {
+  const queryState = useChainsQuery({});
+  const { register, watch, trigger } = useFormContext<WithdrawValues>();
+  const network = watch("network");
+
+  useEffect(() => {
+    (async () => {
+      //validate address on network change
+      await trigger("beneficiary");
+    })();
+  }, [network, trigger]);
+
+  return (
+    <div className="grid gap-3">
+      <span className="font-work font-bold">Select network</span>
+      <QueryLoader
+        queryState={queryState}
+        messages={{
+          loading: "Loading available networks...",
+          empty: "No available networks",
+          error: "Error loading networks",
+        }}
+      >
+        {(chains) => (
+          <Listbox {...register("network")} as="div" className="relative">
+            <Listbox.Button className="grid grid-cols-[1fr_auto] items-center text-left gap-2 w-full md:w-52 p-4 border border-prim rounded">
+              {({ open, value }) => (
+                <>
+                  <span className="truncate max-md:text-sm">
+                    {
+                      chains.find((chain) => chain.chain_id === value)
+                        ?.chain_name
+                    }
+                  </span>
+                  <DrawerIcon
+                    isOpen={open}
+                    className="text-2xl text-gray-d1 dark:text-white"
+                    aria-hidden="true"
+                  />
+                </>
+              )}
+            </Listbox.Button>
+
+            <Listbox.Options className="absolute mt-1 z-10 flex flex-col gap-1 w-full p-1 border border-prim rounded bg-white dark:bg-blue-d6 shadow-lg">
+              <>
+                {chains.map((chain) => (
```

Comment:
@jp-mariano, is withdrawing to `polygon` and `terra` supported now? 
<img width="577" alt="image" src="https://user-images.githubusercontent.com/89639563/223014419-3bd9022f-6869-4675-ab13-0cbbed91fbf1.png">

Reply:
Introduced those two when fetching chains from AWS. In light of recent discussion that we'll move chain-related SOT to webapp, will update the code to remove this fetching and just hardcode values.